pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -1025,9 +1025,11 @@ def process_circuits(
                     quantinuum_circ = circuit_to_qasm_str(
                         c0,
                         header="hqslib1",
-                        maxwidth=32,
-                        # TODO Get maxwidth from self.backend_info once
-                        # https://github.com/CQCL/tket/issues/1445 fixed in pytket
+                        maxwidth=self.backend_info.misc.get(
```

Comment:
Happy to copy this over to the other PR if you want this to be in there. See https://github.com/CQCL/pytket-quantinuum/pull/466

Reply:
Forgot about this.